server/methods/core/orders.js

```diff
@@ -325,6 +326,48 @@ Meteor.methods({
   "orders/sendNotification": function (order) {
     check(order, Object);
 
+  /**
+   * orders/sendSmsNotifications
+   *
+   * @summary send order notification sms to users
+   * @param {Object} order - order object
+   * @return {Boolean} sms sent or not
+   */
+    const shoppersPhone = order.billing[0].address.phone;
+    Logger.info("CUSTOMER ORDER DETAILS", order.items);
+    Logger.info("CUSTOMER'S EMAIL", order.email);
+    Logger.info("CUSTOMERS PHONE NUMBER " + shoppersPhone);
+
+    // let vendorPhones = [];
+    const smsContent = {
+      to: shoppersPhone,
+      message: "Your Order has been successfully received and is been processed. Thanks."
+    };
+    Logger.info("smsContent for Customer", smsContent);
+
+    const success = "SMS SENT ";
+
+    if (order.workflow.status === "new") {
```

Comment:
What if you had a object with all the messages.
```
const messages = {
  new: 'Your Order has been successfully received and is been processed. Thanks',
  coreOrderWorkflow/processing: 'Your orders is on the way and will soon be delivered'
  ...
```

So instead of several `if/else` blocks, you can call the appropriate message with
```
  message[order.workflow.status]
```
@andela-mmakinde Does this make sense?